src/site/content/en/blog/betty-crocker/index.md

```diff
@@ -0,0 +1,340 @@
+---
+title: |
+  Better cooking with the Wake Lock API : A&nbsp;case study with Betty Crocker
+subhead: |
+  After they have implemented the Wake Lock API, cooking site BettyCrocker.com's
+  indicators of purchase intent were about 300% higher for wake lock users compared to all users.
+authors:
+  - thomassteiner
+date: 2020-04-22
+updated: 2020-04-22
+hero: hero.jpg
+alt: Betty Crocker Cake Mix—Coffee Cinnamon.
+description: |
+  Case study of cooking site BettyCrocker.com's experiences with implementing the Wake Lock API.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - case-study
+  - wake-lock
+  - capabilities
+  - fugu
+draft: true
+---
+For nearly a century, Betty Crocker has been America's source for modern cooking instruction
+and trusted recipe development.
+Launched in 1997, their site [BettyCrocker.com](https://www.bettycrocker.com/)
+today receives more than 12&nbsp;million visitors per month.
+After they **implemented the Wake Lock API**, their
+**indicators of purchase intent were about 300% higher**
+for wake lock users compared to all users.
+
+## The retired iOS and Android apps
+
+Released to [much fanfare](https://consumergoods.com/betty-crocker-launches-cookbook-app) in 2014,
+Betty Crocker recently took their native apps out of the Apple App Store and the Google Play Store
+after it had been mostly neglected and deprioritized.
```

Comment:
```suggestion
after they had been mostly neglected and deprioritized.
```

This should be `they` because it's a reference to `native apps` (plural)